OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix inventory

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/417bbb99-4714-49bc-91a2-db2787b92e05)

Eslint fixed:
![CleanShot 2023-11-02 at 18 48 03@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/9e3b64e3-60c1-49e6-a2b6-900088ada9ae)


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. Check if the inventory function is normal as before.

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
